extensions/wikia/AdEngine/js/recovery/helper.js

```diff
@@ -41,11 +44,11 @@ define('ext.wikia.adEngine.recovery.helper', [
 	}
 
 	function isRecoveryEnabled() {
-		return !!(context.opts.sourcePoint && SourcePointTag);
+		return !!(context.opts.sourcePointRecovery && SourcePointTag);
```

Comment:
Is it needed? Variable `context.opts.sourcePointRecovery` is created and used only in JS.